Python/optimizer.c

```diff
@@ -157,6 +157,7 @@ _Py_SetTier2Optimizer(_PyOptimizerObject *optimizer)
     return old == NULL ? -1 : 0;
 }
 
+int executors_created = 0;
```

Comment:
Let's make this a `size_t`, and stick it on the `PyInterpreterState` struct, next to the linked list of executors. We avoid using globals like this since it can break in the presence of multiple interpreters or threads.